packages/jest-snapshot/src/utils.js

```diff
@@ -21,6 +21,24 @@ const naturalCompare = require('natural-compare');
 const getSerializers = require('./plugins').getSerializers;
 
 const SNAPSHOT_EXTENSION = 'snap';
+const SNAPSHOT_VERSION = '1';
+const SNAPSHOT_VERSION_REGEXP = /^\/\/ Jest Snapshot v(.+)/g;
+
+const writeSnapshotVersion = () =>
+  `// Jest Snapshot v${SNAPSHOT_VERSION}`;
+
+const validateSnapshotVersion = (snapshotContents: string) => {
+  const versionTest = SNAPSHOT_VERSION_REGEXP.exec(snapshotContents);
```

Comment:
I guess you are already making sure we only match the beginning of the string with ^. I'm not sure why the regex has "g" though? We only want to find the version in the beginning of the file, once.